### ISSUE TITLE
Org Logo

### ISSUE THREAD

**Issue by ashutosh1919** [ashutosh1919/masterPortfolio#40]
New Logos
![masterportfolio-banner-light](https://user-images.githubusercontent.com/20843596/89031990-431ce400-d351-11ea-8e93-ae57a6f5b8d6.png)

**Comment by remy** [comment]
Ripped off straight from Prettier.

<img width="1287" alt="Screenshot 2020-08-02 at 10 36 02" src="https://user-images.githubusercontent.com/13700/89120177-10d3c980-d4ac-11ea-8bdf-dc842fea1ba7.png">

**Comment by ashutosh1919** [comment]
Yes @remy , already mentioned credit to the project and the designer. Please guide me. I thought since it is opensource, we can use it with proper credits
